examples/tutorials/advanced/cartesian_histograms.py

```diff
@@ -0,0 +1,337 @@
+r"""
+Cartesian histograms
+====================
+
+Cartesian histograms can be generated using the :meth:`pygmt.Figure.histogram`
+method. In this tutorial, different histogram related aspects are addressed:
+
+- using vertical and horizontal bars
+- showing counts and frequency percent
+- adding annotations to the bars
+- showing cumulative values
+- using color and pattern as fill for the bars
+- using overlaid, stacked, and grouped bars
+"""
+
+# sphinx_gallery_thumbnail_number = 4
+
+
+# Import the required packages
+import numpy as np
+import pygmt
+
+###############################################################################
+# Generate random data from a normal distribution:
+
+np.random.seed(100)
+
+# Mean of distribution
+mean = 100
+# Standard deviation of distribution
+stddev = 20
+
+# Create two data sets
+data01 = mean + stddev * np.random.randn(42)
+data02 = mean + stddev * 2 * np.random.randn(42)
+
+
+###############################################################################
+# Vertical and horizontal bars
+# ----------------------------
+# To define the width of the bins the ``series`` parameter has to be specified.
+# The bars can be filled via the ``fill`` parameter with either a color or a
+# pattern (see later in this tutorial). Use the ``pen`` parameter to adjust the
+# outline of the bars.
+
+# Create new figure instance
+fig = pygmt.Figure()
+
+# Create histogram for data01
+fig.histogram(
+    # Define the plot range as a list of xmin, xmax, ymin, ymax
+    # Let ymin and ymax be determined automatically via setting both to zero
```

Comment:
I believe *ymin* is always set to 0 by default.
```suggestion
    # Let ymax be determined automatically by setting it to zero
```

Reply:
Yes, when the same values for _ymin_ and _ymax_ are given, _ymin_ is always set to zero and _ymax_ is determined automatically.

We probably should also update the comment in the Gallery example [Scatter plot with histograms](https://www.pygmt.org/dev/gallery/histograms/scatter_and_histograms.html).

Reply:
I reverted this change in commit https://github.com/GenericMappingTools/pygmt/pull/2445/commits/4e8610af8b9f83a7731e67a9c5c02b1d5112aeed, as for logarithmic projects _ymin_ is not set to zero, please see comment https://github.com/GenericMappingTools/pygmt/pull/2526#discussion_r1180213727 by @seisman.